content/Felles datakatalog/Search/Search/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Search
+weight: 1
+---
+
+The service responsible for handling searches is [fdk-search-service](https://github.com/Informasjonsforvaltning/fdk-search-service), which is described by [this OpenAPI specification](https://raw.githubusercontent.com/Informasjonsforvaltning/fdk-search-service/main/openapi.yaml).
+
+Production endpoint: <https://search.api.fellesdatakatalog.digdir.no/search>
+Demo endpoint: <https://search.api.demo.fellesdatakatalog.digdir.no/search>
+Staging endpoint: <https://search.api.staging.fellesdatakatalog.digdir.no/search>
+
+Simple example using the staging endpoint:
+```Shell
+curl -X POST 'https://search.api.staging.fellesdatakatalog.digdir.no/search' -H 'Content-Type: application/json' -d '{"query":"test"}'
+```
+
+### Boosting
+
+Hits from some fields will be prioritized over others, i.e. a matching hit from the title field will be prioritized over a hit from the description field
+
+### Pagination
+
+All search results will be paginated, it is possible to manipulate the size and page number with the pagination field in the search body.
+
+Example using the pagination field:
+```Shell
+curl -X POST 'https://search.api.staging.fellesdatakatalog.digdir.no/search' -H 'Content-Type: application/json' -d '{"query":"test","pagination":{"size":1,"page":5}}'
+```
+
+### Filtering
+
+It's possible to filter the search result, see SearchFilters in the OpenAPI specification for a list of all possible filters.
```

Comment:
Jeg er usikker på om yaml-filen gir meg nok informasjon. Hvor finner jeg de faktiske filterverdiene jeg kan filtrere på, som de du har med i eksempelet ditt? Jeg kommer til EU-tema, men finnes det en referanse til de faktiske verdiene der, eller er det noe man finner ved å bruke den lenken det henvises til? Jeg er litt noob på dette, men nysgjerrig :)

Reply:
verdiene som er tilgjengelige kommer an på hva vi har høstet, så selv om vi skulle pekt til kodelistene som anbefales for de forskjellige feltene så er det langt fra alle kodene som vil ha treff

Reply:
Enkleste måten å se hva som er tilgjengelig for de forskjellige filtrene er å se på aggregeringen i responsen, jeg kan prøve å få inn noe info om det også

Reply:
576ff7e